tests/test_telegramobject.py

```diff
@@ -238,9 +238,9 @@ def test_pickle(self, bot):
         with pytest.raises(RuntimeError):
             unpickled.get_bot()  # There should be no bot when we pickle TGObjects
 
-        assert unpickled.chat == chat
+        assert unpickled.chat == chat, f"{unpickled.chat._id_attrs} != {chat._id_attrs}"
         assert unpickled.from_user == user
-        assert unpickled.date == date
+        assert unpickled.date == date, f"{unpickled.date} != {date}"
```

Comment:
these changes are not really related right? just double checking, we can keep them either way

Reply:
yeah not related. They had failed during testing, so wanted to find out what the problem was.